easybuild/easyconfigs/b/BEEF/BEEF-0.1.1-r16-intel-2015a.eb

```diff
@@ -0,0 +1,33 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Benjamin P. Roberts
+# New Zealand eScience Infrastructure
+# The University of Auckland, Auckland, New Zealand 
+
+easyblock = 'ConfigureMake'
+
+name = 'BEEF'
+version = '0.1.1-r16'
+
+homepage = 'http://suncat.stanford.edu/facility/software/functional'
+description = """BEEF is a library-based implementation of the Bayesian
+Error Estimation Functional, suitable for linking against by Fortran-
+or C-based DFT codes. A description of BEEF can be found at
+http://dx.doi.org/10.1103/PhysRevB.85.235149."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'openmp': False, 'usempi': False}
+
+source_urls = ["svn://suncatls1.slac.stanford.edu/beef"]
+#for url in source_urls:
+#    svn co -r 16 url
+#    tar --exclude-vcs -cjvf "beef-0.1.1-r16.tar.bz2" "beef"
+#sources = [SOURCELOWER_TAR_BZ2]
```

Comment:
I'm not entirely sure. Does Easybuild allow for checking out SVN sources directly?